runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -70,6 +70,9 @@ function M.on_inlayhint(err, result, ctx, _)
     return
   end
 
+  if not vim.api.nvim_buf_is_valid(bufnr) then
```

Comment:
I feel like this is not the right place to change. If the buffer is still being tracked in the internal state table then it must have missed some unloading/deletion buffer signal (and that's what we should change instead).

Reply:
Is there a better way to solve this problem? :(